src/App.css

```diff
@@ -0,0 +1,90 @@
+body {
+    padding: 0px;
+    margin: 0px;
+    font-family: 'Open sans';
+}
+h1 {
+    margin: 0px;
+    font-weight: 700;
+    color: #000000;
+    line-height: 1.2em;
+    font-size: 2.1em;
```

Comment:
Attention  à l'usage de l'em, l'em est très variables. il dépend du parent, il est préférable d'utiliser les rem  afin d'adapter la proportion de font-size en fonction de la font paramétré par l'utilisateur  sur son navigateur ! :) 
là dessus plusieurs écoles se battent, je te laisse décider :P 
au cas où un petit [article](https://j.eremy.net/confused-about-rem-and-em/)

Reply:
Merci pour l'article, je vais checker ça !